src/components/Demo/DemoCreate/DemoCreateForm/data.tsx

```diff
@@ -0,0 +1,36 @@
+export const data = {
+  cooId: "WBC 208897",
+
+  //needs to be set when issued
+  issueDate: "Mon 5 July 2021",
+  issueTime: "2:15 PM",
```

Comment:
yup understand that, should we just create the date and the time here instead of using a fix date as the default?